python/paddle/fluid/contrib/slim/tests/slim_int8_mkldnn_post_training_quantization.md

```diff
@@ -0,0 +1,82 @@
+
+## 0. Install PaddlePaddle
+Follow PaddlePaddle [installation instruction](https://github.com/PaddlePaddle/models/blob/develop/PaddleCV/image_classification/README_ngraph.md) to install PaddlePaddle. If you [build from source](https://github.com/PaddlePaddle/FluidDoc/blob/develop/doc/fluid/beginners_guide/install/compile/compile_Ubuntu_en.md), please use the following cmake arguments.
+
+```
+cmake .. -DCMAKE_BUILD_TYPE=Release -DWITH_GPU=OFF -DWITH_MKL=ON -DWITH_MKLDNN=ON  -DWITH_TESTING=ON  -WITH_FLUID_ONLY=ON  -DWITH_INFERENCE_API_TEST=ON -DON_INFER=ON -DWITH_SLIM_MKLDNN_FULL_TEST=ON
+```
+
+Note: MKL-DNN and MKL are required.
+
+## 1. Accuracy benchmark
+
+We provide the results of accuracy measured on Intel(R) Xeon(R) Gold 8280 on single core.
+
+   >**I. Top-1 Accuracy on Intel(R) Xeon(R) Gold 8280**
+
+| Model  | Dataset  | FP32 Accuracy  | INT8 Accuracy  | Accuracy Diff  |
+| :------------: | :------------: | :------------: | :------------: | :------------: |
+| GoogleNet    | Full ImageNet Val  |  70.50% |  70.20% |  0.30% |
+| MobileNet-V1 | Full ImageNet Val  |  70.78% |  70.36% |  0.42% |
+| MobileNet-V2 | Full ImageNet Val  |  71.90% |  71.57% |  0.33% |
+| ResNet-101   | Full ImageNet Val  |  77.50% |  77.53% |  -0.03%|
+| ResNet-50    | Full ImageNet Val  |  76.63% |  76.48% |  0.15% |
+| VGG16        | Full ImageNet Val  |  72.08% |  72.01% |  0.07% |
+| VGG19        | Full ImageNet Val  |  72.56% |  72.56% |  0.00% |
+
+Notes:
+* CPU turbo off.
+
+## 2. Commands to reproduce the above accuracy benchmark
+* #### Full dataset (Single core)
+   * ##### Download full ImageNet Validation Dataset
+```bash
+cd /PATH/TO/PADDLE/build
+python ../paddle/fluid/inference/tests/api/full_ILSVRC2012_val_preprocess.py
+```
+The converted data binary file is saved by default in ~/.cache/paddle/dataset/int8/download/int8_full_val.bin
+
+   * ##### GoogleNet Full dataset benchmark
+```bash
+cd /PATH/TO/PADDLE/build/python/paddle/fluid/contrib/slim/tests/
+python ./test_mkldnn_int8_quantization_strategy.py --infer_model /PATH/TO/PADDLE/build/third_party/inference_demo/int8v2/googlenet/model --infer_data /PATH/TO/CONVERTED/int8_full_val.bin --warmup_batch_size 100 --batch_size 1
+```
+
+   * ##### MobileNet-v1 Full dataset benchmark
+```bash
+cd /PATH/TO/PADDLE/build/python/paddle/fluid/contrib/slim/tests/
+python ./test_mkldnn_int8_quantization_strategy.py --infer_model /PATH/TO/PADDLE/build/third_party/inference_demo/int8v2/mobilenet/model --infer_data /PATH/TO/CONVERTED/int8_full_val.bin --warmup_batch_size 100 --batch_size 1
```

Comment:
I tried to run this script on 2699v3. It seems taking long time (>10mins) without any in-progress indicator. Just staying on below prompt:
2019-06-07 22:17:11,436-INFO: --- INT8 prediction start ---

It's not user friendly. Can we at least indicate a rough time in this doc?

Reply:
done with adding in `The above commands will cost maybe several hours in the prediction stage (include int8 prediction and fp32 prediction) since there have 50000 pictures need to be predicted in int8_full_val.bin` the doc

Reply:
Could you add some log in the script, i.e every predicts 100 pictures, print some logs. For example, `test_calibration.py` print logs every 100 pictures.
https://github.com/PaddlePaddle/Paddle/blob/e4a52e082fe84a4713d05370f077eeda413a870c/python/paddle/fluid/contrib/tests/test_calibration_resnet50.py#L266-L268

Reply:
done with PR [17920](https://github.com/PaddlePaddle/Paddle/pull/17920)